packages/block-library/src/heading/heading-level-icon.js

```diff
@@ -3,6 +3,24 @@
  */
 import { Path, SVG } from '@wordpress/components';
 
+/** @typedef {import('@wordpress/element').WPComponent} WPComponent */
+
+/**
+ * HeadingLevelIcon props.
+ *
+ * @typedef WPHeadingLevelIconProps
+ *
+ * @property {number}   level     The heading level to show an icon for.
+ * @property {?boolean} isPressed Whether or not the icon should appear pressed; default: false.
```

Comment:
is this how we mark optional types, I thought it was  more `boolean=`

Reply:
I'm using the `?xxx` format because [these docs](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#nullable-undefined-and-void-types) said to use it for nullable types.